src/crypto-providers/ledgerCryptoProvider.ts

```diff
@@ -253,6 +292,23 @@ export const LedgerCryptoProvider: () => Promise<CryptoProvider> = async () => {
     }
   }
 
+  const prepareTtl = (ttl?: number) => {
+    if (!ttl && !isFeatureSupportedForVersion(LedgerCryptoProviderFeature.ALLEGRA)) {
```

Comment:
ttl equal to zero would trigger a false error (although such transaction would be practically invalid anyway)

I'd substitute it with `ttl == null` check

Reply:
Fixed